src/index.html

```diff
@@ -6,9 +6,71 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_items">
+            <a href="#" class="nav_link nav_link_1 is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_items">
+            <a href="#" class="nav_link nav_link_2">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_items">
+            <a href="#" class="nav_link nav_link_3">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav_items">
+            <a href="#"
+            class="nav_link nav_link_4"
+            data-qa="hover"
+            >
```

Comment:
apply for cases below the same fixes
```suggestion
          <li class="nav_item">
            <a href="#"
              class="nav_link"
              data-qa="hover"
            >
```